### PR TITLE
#162163220 SetUp folder/file structure for redux

### PR DESCRIPTION
**Pull request by Nta1e**
**What does this PR do?**
This Pull Request sets up the folder/file structure for `Redux`
**Description of Task to be completed?**
We anticipate `Authors Heaven` to be a huge project so we need to integrate `Redux` into our front-end `React` app for proper state management and easy maintainability
**How should this be manually tested?**
- Install the `Redux Devtools` chrome extension
- Clone the repo
- Check out the `ch-setup-redux-162163220` branch
- run `npm` install
- run `npm start` to start the development server
- confirm that the `Redux Devtools` extension is activated meaning the application is using redux

**What are the relevant pivotal tracker stories?**
#162163220

### CODE REVIEW

**Review by joshtrigger — APPROVED** [review]

**Review by bibangamba — APPROVED** [review]

**Review by JaymesKat — CHANGES_REQUESTED** [review]
@Nta1e could you adjust your commit message body to have an imperative/present tense for tasks done? [reference](https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions#message-body )

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```
Comment (JaymesKat) [inline .gitignore]: @Nta1e Why did you choose to add the package-lock.json to the .gitignore? It is needed.

Comment (Nta1e) [reply .gitignore]: ok let me remove it

Comment (Nta1e) [reply .gitignore]: @JaymesKat I have implemented the feedback

**Review by Kasulejoseph — APPROVED** [review]

**Review by JaymesKat — COMMENTED** [review]
@Nta1e resolve conflicts

**Comment by Nta1e** [comment]
@JaymesKat I have resolved the conflicts

**Review by JaymesKat — APPROVED** [review]
@Nta1e Good job.

**Pull request merged**